find.go

```diff
@@ -130,6 +180,15 @@ outer:
 				resp.MultihashResults[0].ProviderResults = append(resp.MultihashResults[0].ProviderResults, pr)
 			}
 		}
+
+		if len(prov.EncMultihashResults) > 0 {
+			if !bytes.Equal(resp.EncMultihashResults[0].Multihash, prov.EncMultihashResults[0].Multihash) {
+				// weird / invalid.
```

Comment:
Consider writing what it is that's weird; if we are leaving code comments we want them to be as objective as possible :)

Reply:
It's not mine, borrowed from here: https://github.com/ipni/indexstar/blob/main/find.go#L120 :) Will remove